### ISSUE TITLE
Caption Endpoints need to be restored/redirected

### ISSUE THREAD

**Issue by kabel** [unl/UNL_MediaHub#272]
The endpoints for providing SRT captions has been removed, but I have found instances of people still using that endpoint (currently on the home page). The URL should be restored and redirect to the VTT endpoint.

Also, the VTT should work in a similar manner and not require a query string parameter to function. If the variable is needed by the app, it should try to guess it based on the given media id.

**Comment by mfairchild365** [comment]
Should the home page video be updated to use the embed code? We should look
at how we want people to use videos from media hub. My understanding was
that if someone wanted to use a mediahub video on their site, they need to
use the embed code, as that allows us to ensure the embedded video is
always up to date with the latest code. By having customized video players
using media hub, we are hurting learned UI (searchable captions) and
incurring more technical debt.

**Comment by mfairchild365** [comment]
I resolved this in this commit: https://github.com/unl/UNL_MediaHub/commit/1f68daf07e3556cc93abf74f89e958c0078c4aef

This is a slower method of getting captions (more queries against the DB) and does not support multiple languages. My opinion is that this method of getting captions should be deprecated and if someone wants to use captions and videos in a custom way, they should download the captions and host them elsewhere. 

I've added support back because I didn't realize that these endpoints were being used this way, and I didn't communicate those endpoints were being removed.
